### PR TITLE
Ref Guide - Orgs

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10587

Moves content (inc images) to new chapter in Admin Guide, then references it. 

New chapter will require editing and updated screenshots, not yet done.

### CODE REVIEW

File: modules/administration/pages/organizations.adoc [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+image::admin_list_organizations.png[scaledwidth=80%]
+
+If you click the name of an organization, the Organization Details page appears.
+
+
+
+[[s2-sattools-org-details-details]]
+== Organization Details
+
+The menu:Organization[Organization Details] page lists the details of the selected organization.
```
Comment (keichwa) [inline modules/administration/pages/organizations.adoc]:
```suggestion
The organization details page lists the details of the selected organization.
```

File: modules/administration/pages/organizations.adoc [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+====
+These parameters will only have an effect when Content Staging is enabled for the targeted Organization.
+====
+
+
+
+[[s2-sattools-org-details-states]]
+== Organization Details > States
+
+From the menu:Admin[Organizations > States] page you can assign State to all systems in an organization.
```
Comment (keichwa) [inline modules/administration/pages/organizations.adoc]:
```suggestion
From the menu:Admin[Organizations > States] page you can assign State to all clients in an organization.
```

File: modules/administration/pages/organizations.adoc [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+For example, this way it is possible to define a few global security policies or add a common admin user to all machines.
```
Comment (keichwa) [inline modules/administration/pages/organizations.adoc]:
```suggestion
For example, this way it is possible to define a few global security policies or add a common admin user to all clients.
```

**Comment by Loquacity** [comment]
Yeah, I was going to edit in a new PR ;)

**Pull request merged**